package/com.unity.formats.usd/Tests/Editor/EditorReImportAnalyticsTests.cs

```diff
@@ -0,0 +1,149 @@
+            Assert.IsTrue(importUsdEventMsg.Succeeded, "Expected True for import Successful");
+            Assert.IsFalse(importUsdEventMsg.IncludesMaterials, "Expected Includes Materials to be False initially");
+
+            importedObject.GetComponent<UsdAsset>().m_materialImportMode = MaterialImportMode.ImportPreviewSurface;
+            importedObject.GetComponent<UsdAsset>().Reload(false);
+
+            AnalyticsEvent reImportEvent = null;
+            yield return (WaitForUsdAnalytics<UsdAnalyticsEventReImport>(UsdAnalyticsTypes.ReImport, (AnalyticsEvent waitedEvent) => {
+                reImportEvent = waitedEvent;
+            }));
+
+            Assert.IsNotNull(reImportEvent);
+            var reImportUsdEventMsg = ((UsdAnalyticsEventReImport)reImportEvent.usd).msg;
+
+            Assert.IsTrue(reImportUsdEventMsg.Succeeded, "Expected True for re-import Successful");
+            Assert.Greater(reImportUsdEventMsg.TimeTakenMs, 0, "Expected Time Taken MS should be greater than 0 for Re-Import");
+            Assert.IsTrue(reImportUsdEventMsg.IncludesMaterials, "Expected Includes Materials to be True on ReImport with ImportPreviewSurface setting for Material");
+        }
+
+        [UnityTest]
+        [Retry(3)]
+        // ReImport doesnt have different values for the following Includes: PointInstancer, Meshes, Skel
+        public IEnumerator OnValidReImportAndImportPropertiesAreEqual_AnalyticsAreSent([ValueSource("testFileGUIDs")] string testFileGUID)
+        {
+            m_scene = OpenUSDGUIDAssetScene(testFileGUID, out _);
+
+            var importedObject = ImportHelpers.ImportSceneAsGameObject(m_scene);
+            AnalyticsEvent importEvent = null;
+            yield return (WaitForUsdAnalytics<UsdAnalyticsEventImport>(UsdAnalyticsTypes.Import, (AnalyticsEvent waitedEvent) => {
+                importEvent = waitedEvent;
+            }));
+
+            Assert.IsNotNull(importEvent);
+
+            var importUsdEventMsg = ((UsdAnalyticsEventImport)importEvent.usd).msg;
+
+            importedObject.GetComponent<UsdAsset>().m_materialImportMode = MaterialImportMode.ImportPreviewSurface;
+            importedObject.GetComponent<UsdAsset>().Reload(false);
+
+            AnalyticsEvent reImportEvent = null;
+            yield return (WaitForUsdAnalytics<UsdAnalyticsEventReImport>(UsdAnalyticsTypes.ReImport, (AnalyticsEvent waitedEvent) => {
+                reImportEvent = waitedEvent;
+            }));
+
+            Assert.IsNotNull(reImportEvent);
+            var reImportUsdEventMsg = ((UsdAnalyticsEventReImport)reImportEvent.usd).msg;
+
+            Assert.IsTrue(reImportUsdEventMsg.Succeeded, "Expected True for re-import Successful");
+            Assert.Greater(reImportUsdEventMsg.TimeTakenMs, 0, "Expected Time Taken MS should be greater than 0 for Re-Import");
+            Assert.AreEqual(importUsdEventMsg.IncludesSkel, reImportUsdEventMsg.IncludesSkel);
```

Comment:
The point of this specific test case is that for the initial import and re-import the properties are equal
If theres a problem with the initial import, it would have been caught with the import tests, and if theres a problem with the re-import it would have been caught in the re-import specific tests
This testcase is to make sure that on re-import, some properties do not change for files that have meshes, skel and